src/data/polynomial/div.lean

```diff
@@ -19,11 +19,11 @@ noncomputable theory
 open_locale classical big_operators
 
 open finset
-
 namespace polynomial
 universes u v w z
 variables {R : Type u} {S : Type v} {T : Type w} {A : Type z} {a b : R} {n : ℕ}
 
+/-
```

Comment:
If this commented-out section has been moved to the new `inductions.lean` file, we can just delete it.

Reply:
Done!